group_vars/all

```diff
@@ -152,7 +152,7 @@ pkg_version_pandas: 0.25.3
 pkg_version_pcaspy: 0.7.2
 pkg_version_psutil: 5.6.7
 pkg_version_pyepics: 3.4.0
-pkg_version_pyqtgraph: 0.10.0
+pkg_version_pyqtgraph: 0.11.0
```

Comment:
@xresende and @anacso17, I think we need to ask someone from the Controls group to review this PR, since they also use `pyqtgraph` and `pydm`

Reply:
It is true. I added @carneirofc.

Reply:
thanks!

Reply:
Guys, @carneirofc is on vacation rs
I talked to him last week and talked to @ ito-rafael this week about this update!
They prepared the pydm_opi v1.4.5 version for this (I already added it) !

Reply:
ok @anacso17 . I am merging this PR in...